src/io/ioexternalbus.h

```diff
@@ -39,11 +42,24 @@ class IOExternalBus : public IOBase {
 protected:
     virtual void parameterChanged(unsigned) override;
 
+private slots:
+    void connectButtonTriggered();
+    // void sktConnected();
+    // void sktRead();
+
 private:
+    uint32_t ByteSize;
+    int32_t send_cmd(const uint32_t cmd, const uint32_t payload_size = 0, const char* payload = nullptr);
```

Comment:
`size_t` for payload size? + payload as QByteArray? (like above)

Reply:
changed